src/jit/codegenarm.cpp

```diff
@@ -778,7 +778,7 @@ void                CodeGen::genCodeForBBlist()
                 }
             }
         }
-#endif _TARGET_AMD64_
+#endif //_TARGET_AMD64_
```

Comment:
What's the best solution here? Revert and suppress the warning (this warning is enabled in all cases currently), or to commit these fixes, it's as this code was never built on Linux so it does somewhat come under the get it building.